classes/Layout.php

```diff
@@ -20,32 +20,45 @@ public function __construct(
         public string $footer,
     ) {}
 
-    public function renderBeforeContentHTML(System $system, ?User $player, string $page_title, bool $custom_page = false): void {
+    public function renderBeforeContentHTML(System $system, ?User $player, string $page_title, bool $render_header = true, bool $render_sidebar = true, bool $render_topbar = true, bool $render_content = true): void {
```

Comment:
There's enough params we could move to each on its own line for readability
```suggestion
    public function renderBeforeContentHTML(
      System $system, 
      ?User $player, 
      string $page_title, 
      bool $render_header = true, 
      bool $render_sidebar = true, 
      bool $render_topbar = true, 
      bool $render_content = true
    ): void {
```